lib/localvore_sdk.ex

```diff
@@ -22,7 +29,7 @@ defmodule LocalvoreSdk do
   defp build_url(query, resource), do: resource <> "?" <> query
 
   defp domain do
-    Application.get_env(:localvore_sdk, :api_url, @default_api_url)
+    Application.get_env(:localvore_sdk, :api_url, @api_url)
```

Comment:
This change has brought this function to my attention, is there a reason we're using functions for this vs something like:

```elixir
@default_api_url "https://api.localvoretoday.com"
@default_api_version "2016-10-28"

@domain Application.get_env(:localvore_sdk, :api_url, @default_api_url)
@version Application.get_env(:localvore_sdk, :api_version, @default_api_version)
```

On a separate note, I think the `default` prefix is nice-to-have so it's clear it's a default.